tiledb/sm/storage_manager/storage_manager.cc

```diff
@@ -2246,32 +2245,28 @@ Status StorageManager::load_group_metadata(
   const auto& group_metadata_to_load = group_dir->group_meta_uris();
```

Comment:
We might be able to easily convert load_group_metadata to return void and throw instead of returning Status.

Reply:
done